services/QuillLMS/db/migrate/20220920190724_drop_old_concept_results.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DropOldConceptResults < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :concept_results, :old_concept_result_id
```

Comment:
This one should be remove_reference vs remove_column since we want to undo this [operation](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/db/migrate/20220707155016_create_concept_result.rb#L10)

Reply:
My thinking is: eventually we want to drop the column, so might as well run `remove_column` now, to remove both the column and the reference in one fell swoop.

Reply:
If `remove_column` runs both then ignore my comment.